powershell/VstsTaskSdk/VstsTaskSdk.psd1

```diff
@@ -9,7 +9,7 @@
     PowerShellVersion = '3.0'
     FunctionsToExport = '*'
     CmdletsToExport = ''
-    VariablesToExport = 'IssueSources'
+    VariablesToExport = '*'

```

Comment:
Not sure the implication of changing this to "*" -- would it be safer to make more narrow?

Reply:
Even though it is specified here that we export all members it isn't so because we explicitly export members in the psm1 file. It's completely safe. The same works for function export.
https://github.com/microsoft/azure-pipelines-task-lib/blob/4ee6f874499c3b5cfc599871f91cb52b3a3fcd23/powershell/VstsTaskSdk/VstsTaskSdk.psm1#L98-L100